v22.1/common-errors.md

```diff
@@ -30,7 +30,7 @@ This message indicates a client is trying to connect to a node that is either no
 To resolve this issue, do one of the following:
 
 - If the node hasn't yet been started, [start the node](cockroach-start.html).
-- If you specified a [`--listen-addr` and/or a `--advertise-addr` flag](cockroach-start.html#networking) when starting the node, you must include the specified IP address/hostname and port with all other [`cockroach` commands](cockroach-commands.html) or change the `COCKROACH_HOST` environment variable.
+- If you specified a [`--listen-addr` and/or a `--advertise-addr` flag](cockroach-start.html#networking) when starting the node, you must include the specified IP address/hostname and port with all other [`cockroach` Commands Overview](cockroach-commands.html) or change the `COCKROACH_HOST` environment variable.
```

Comment:
similar to above, this appears to be a grammar error introduced by search and replace.  the former text was correct IMO, this is not